src/js/_enqueues/wp/heartbeat.js

```diff
@@ -721,9 +722,9 @@
 		 *
 		 * @memberOf wp.heartbeat.prototype
 		 *
-		 * @param {string|number} speed Interval: 'fast' or 5, 15, 30, 60, 120.
+		 * @param {string|number} speed Interval: 'fast' or integer between 1 and 3600 (seconds).
 		 *                              Fast equals 5.
-		 * @param {string}        ticks Tells how many ticks before the interval reverts
+		 * @param {number}        ticks Tells how many ticks before the interval reverts
 		 *                              back. Used with speed = 'fast' or 5.
```

Comment:
Unrelated but probably worth adding a note that it needs to be between 1 & 30.

Reply:
Sure, sounds good.